src/main/java/at/hannibal2/skyhanni/config/features/inventory/GetFromSackConfig.java

```diff
@@ -23,4 +24,15 @@ public class GetFromSackConfig {
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean superCraftGFS = true;
+
+    @Expose
+    @ConfigOption(name = "GfS Default", desc = "When enabled, using §e/gfs §7without specifying an amount will give a default amount of the item.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean gfsGetDefault = false;
+
+    @Expose
+    @ConfigOption(name = "GfS Default Amount", desc = "The default amount of items to get when §e/gfs §7is used without amount.")
```

Comment:
```suggestion
    @ConfigOption(name = "GfS Default Amount", desc = "The default amount of items to get when §e/getfromsacks §7is used without amount.")
```

Reply:
Dont think thats a good one. Just makes the text longer for no reason

Reply:
fair, i just thought it would be better to be consistent with the other option that has the command